.nvmrc

```diff
@@ -1 +1 @@
-14
+v18.14
```

Comment:
The engines is the absolute minimum.

18.15 is much better (in the implementation of the `node:test` framework), than 18.14. So while I don't use `.nvmrc` myself, would suggest at least that version for an auto-selector.

Reply:
Sounds great, thanks for the suggestion. Updated!

Reply:
After briefly reading the LTS notes on 18.15 I definitely agree.